0005-on-chain-voting-transaction-specification.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+// PeerAssets voting protocol specification
+// written by peerchemist <peerchemist@protonmail.ch>
+
+// Identity message
+message Vote {
+
+  // voting protocol version number
+  uint32 version = 1;
+
+  // Modes for card voting
+  enum MODE {
```

Comment:
Change mode codes to
```
    NONE = 0x00
    SIMPLE = 0x01; // One vote per transaction
    WEIGHT_CARD_DAYS = 0x02; // Weightnig vote with PA card days
    WEIGHT_CARD_BALANCE = 0x04; // Weightning vote with PA card balance
```
Because 0x00 is no bit set, this allows SIMPLE mode to be combined with others.